doc/en/user/source/production/config.rst

```diff
@@ -10,6 +10,11 @@ Logging may visibly affect the performance of your server. High logging levels a
 
 You can change the logging level in the :ref:`config_globalsettings_log_profile`.  You will want to choose the ``PRODUCTION`` logging configuration, where only problems are written to the log files.
 
+Logging configuration hardening
+-------------------------------
+
+For production systems, it is advised to set the log location via the environment variable. This prevents overriding it via the administration GUI or REST API, providing better separation of responsibility between those setting up and controlling the actual machine, and those configuring the GeoServer data, styles and the like. Follow :ref:`overriding_the_log_location_setup_in_the_geoserver_configuration` for more information
+
```

Comment:
Thanks, that is clearly written.

There are actually several ways (environmental variable, java system property, or web application parameter) to set these things. I know most system admins would just use the environmental variable, but there may still be someone configuring web application parameters in tomcat (hint I am that someone).

aside: I believe there are a number of settings that can be defined in this manner that all could be used if one wished to enforce a strong separation of concerns between those running the machine and the REST / GUI interface.